content/en/docs/quickstart.md

```diff
@@ -71,13 +71,15 @@ Use `notation sign` to sign the container image.
 notation sign $IMAGE
 ```
 
-By default, the signature format is `JWS`. Use `--signature-format` to use `COSE` signature format.
+By default, the signature format is `JWS`. Use `--signature-format` to use [COSE](https://cose-wg.github.io/cose-spec/) signature format.
```

Comment:
Let's use https://datatracker.ietf.org/doc/html/rfc8152 for COSE's hyperlink.

Reply:
done